### PR TITLE
Update meta.yaml for new package version from CRAN

### PR DESCRIPTION
**Pull request by melpetera**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by melpetera** [comment]
@conda-forge-admin, please rerender

**Comment by melpetera** [comment]
@conda-forge/r  could you review my PR please?

**Comment by melpetera** [comment]
@conda-forge-admin, please ping conda-forge/r

**Comment by melpetera** [comment]
Could you review my PR please?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+{% set version = "1.0.0" %}
 
 package:
   name: r-w4mrutils
   version: "{{ version }}"
 
 source:
-  url: https://github.com/workflow4metabolomics/W4MRUtils/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: b6c2f7b3aae2bb88a8f27db40596a76bcd5f84b2d1324f8147787b070087c037
+  url: https://cran.r-project.org/src/contrib/W4MRUtils_{{ version }}.tar.gz
```
Comment (mfansler) [inline recipe/meta.yaml:8]: Use Jinja variable and pre-emptively include archive searching:

```suggestion
  url:
    - {{ cran_mirror }}/src/contrib/W4MRUtils_{{ version }}.tar.gz
    - {{ cran_mirror }}/src/contrib/Archive/W4MRUtils/W4MRUtils_{{ version }}.tar.gz
```

File: recipe/LICENSE.md
Comment (mfansler) [inline recipe/LICENSE.md:1]: This isn't necessary. Instead, the `license_file` should be set to pick up the shared copy from R, i.e.,

```
license_file: {{ environ["PREFIX"] }}/lib/R/share/licenses/AGPL-3
```

I.e., delete this file and make this change in `meta.yaml`.

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this PR. There are some changes suggested.

File: recipe/LICENSE.md
Comment (melpetera) [reply recipe/LICENSE.md:1]: Done in 96c47bf31c2d1498e6ad1494ef106f0feae9e4aa and ebf3af55cf885c42a433a43d257c702b49ccb1c6

Thank you for the review.

**Comment by melpetera** [comment]
Thank you for the review.
I hope I did the right modifications.

**Review by mfansler — APPROVED** [review]

**Pull request merged**